pkg/minikube/cluster/cluster.go

```diff
@@ -198,7 +198,7 @@ func (m *MachineConfig) CacheMinikubeISOFromURL() error {
 	defer response.Body.Close()
 
 	if response.StatusCode != http.StatusOK {
-		return fmt.Errorf("Received %d response from %s while trying to download minikube.iso", response.StatusCode, m.MinikubeISO)
+		return fmt.Errorf("Received %d response from %s while trying to download minishift.iso", response.StatusCode, m.MinikubeISO)
```

Comment:
This is in flux and in fact dynamic. The default will be _minishift-b2d.iso_, but it is configurable. So there is more needed than just a typo fix.